epub33/rs/index.html

```diff
@@ -1761,7 +1777,7 @@ <h2>Security and Privacy</h2>
 			<p>The particularity of an <a>EPUB Publication</a> is its structure. The EPUB format provides a means of
 				representing, packaging, and encoding structured and semantically enhanced Web content — including HTML,
 				CSS, SVG, and other resources — for distribution in a single-file container. This specification does not
-					<em>add</em> any new features, APIs, etc. This also means that, essentially, the security and
+					<em>add</em> any new features, complex APIs, etc. This also means that, essentially, the security and
```

Comment:
Aren't most APIs pretty complex to the average author? ;)

It's not clear what this addition is getting at, though? (I'm assuming epubReadingSystem object?)

The sentence seems inaccurate taken more broadly, though. We've added the SSML attributes, the (now deprecated but still available) epub:* elements, eRS object, etc.

Reply:
I is indeed referring to `epubReadingSystem`, meaning that we do define a very simple API. Ie, the statement not _adding_ an API is wrong. But the API is dead simple.

I am not sure the epub:* elements are relevant. I believe that statement refers to, essentially, Javascript API-s (those are the ones that are relevant for the origin issue).